glow/event_log.go

```diff
@@ -0,0 +1,112 @@
+package glow
+
+// This file contains a simple event logger, designed to collect a limited
+// number of timestamped logs and dump them to file or terminal.
+
+import (
+	"container/list"
+	"fmt"
+	"strings"
+	"sync"
+	"time"
+)
+
+// Event.
+type event struct {
+	ts   time.Time
+	line string
+}
+
+// Options for the event log.
+type EventLogOptions struct {
+	Expiry     time.Duration // Older events will be removed. Default is no expiry.
+	LimitBytes int           // Limit the size of events to collect. Default is not to collect events.
```

Comment:
default to collect no events feels like a bad default to me. Maybe return an error that suggests a limit of 10e6?

Reply:
Code will require non-zero parameters